src/main/java/uk/gov/companieshouse/authcodenotification/service/PrivateDataRetrievalService.java

```diff
@@ -32,12 +34,23 @@ public OverseasEntityDataApi getOverseasEntityData(String requestId, String comp
                     .execute()
                     .getData();
 
-            ApiLogger.infoContext(requestId, "Successfully retrieved overseas entity data from database",  logDataMap.getLogMap());
+            ApiLogger.infoContext(requestId, "Successfully retrieved overseas entity data from database", logDataMap.getLogMap());
             return overseasEntityDataApi;
-        } catch (URIValidationException | IOException e) {
-            var message = "Error retrieving overseas entity data from database";
-            ApiLogger.errorContext(requestId, message, e, logDataMap.getLogMap());
-            throw new ServiceException(e.getMessage(), e);
+        } catch (ApiErrorResponseException e) {
+            if (e.getStatusCode() == 404) {
+                var message = "Http exception status: " + e.getStatusCode();
```

Comment:
V minor, but I would capitalise 'HTTP' in the message.

Reply:
Changed